inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/manager/enums/WorkloadState.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.workload.dynamic.manager.enums;
+
+import de.tudarmstadt.ukp.clarin.webanno.support.PersistentEnum;
+
+public enum WorkloadState implements PersistentEnum
+{
+    WORKLOAD_MONITORING("WORKLOAD_MONITORING"),
+    DEFAULT_MONITORING("DEFAULT_MONITORING");
```

Comment:
Is it possible to move enum somewhere else, e.g. the central/default workflow place? The dynamic workflow module should not define the default one. Also, I think @reckart  does not like enums but would prefer string constants.